com-scrape/src/generator.rs

```diff
@@ -53,6 +53,7 @@ pub struct Generator {
 
 impl Generator {
     pub fn new() -> Self {
+        clang_sys::load().unwrap();
```

Comment:
Would you be able to move this into `generate()`, and wrap it in `if !clang_sys::is_loaded()`?